parser_test.go

```diff
@@ -2040,7 +2040,7 @@ func (s *testParserSuite) TestType(c *C) {
 		// for bit
 		{"select 0b01, 0b0, b'11', B'11'", true, "SELECT b'1',b'0',b'11',b'11'"},
 		{"select 0B01", false, ""},
-		{"select 0b21", false, ""},
+		// {"select 0b21", false, ""}, 0b21 is an identifier, the statement could parse.
```

Comment:
🤔 `0B01` is also an identifier. I think both lines 2042 and 2043 should have the same treatment (either both can be parsed or both need to be commented out).

Reply:
Fixed